tests/test_e2e_31_of_lldp_loop_detection.py

```diff
@@ -0,0 +1,129 @@
+import requests
+from tests.helpers import NetworkTest
+import time
+
+CONTROLLER = '127.0.0.1'
+KYTOS_API = 'http://%s:8181/api/kytos' % CONTROLLER
+
+
+class TestE2EOfLLDPLoopDetection:
+    net = None
+
+    @classmethod
+    def setup_class(cls):
+        cls.net = NetworkTest(CONTROLLER, topo_name='looped')
+        cls.net.start()
+        cls.net.restart_kytos_clean()
```

Comment:
It seems that we don't need a restart during the setup class, and to make sure this will be more deterministic during setup could you add these two lines to ensure switches will be connected?

```
        cls.net.wait_switches_connect()
        time.sleep(10)
```

Reply:
You are right.

Reply:
In order for the tests not to fail, it is necessary to set the -E option to true so that the loops are detected. Therefore, I must add 'cls.net.start_controller(enable_all=True)'.

Reply:
Sounds good.